modules/tgs/default.nix

```diff
@@ -81,6 +81,9 @@ in {
       ]
     );
   };
+  systemd.services.tgstation-server = {
+    After = ["mysql.service"];
```

Comment:
```suggestion
    Wants = ["mariadb.service"];
```

Reply:
Are you absolutely sure it's mariadb? I can't connect to the servers atm